Plutarch/ByteString.hs

```diff
@@ -74,6 +78,23 @@ pbyteStr = pconstant
 pconsBS :: Term s (PInteger :--> PByteString :--> PByteString)
 pconsBS = punsafeBuiltin PLC.ConsByteString
 
+-- | Case matching on bytestring, as if a list.
+pelimBS ::
```

Comment:
I don't think we should export this function. It's not directly necessary for Plutarch, nor is it even a good abstraction to use.